<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="8" hostname="HAT" failures="1" tests="12" name="Test" time="3.03" errors="0" timestamp="2024-02-27T14:15:14 CST">
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@BeforeMethod beforeMethod" time="1.849">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.testcases.testNg.TestNgBase.beforeMethod(TestNgBase.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod beforeMethod -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@AfterMethod afterMethod" time="-1.709064913625E9">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@BeforeMethod beforeMethod" time="-1.709064913629E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod beforeMethod -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@AfterMethod afterMethod" time="-1.709064913634E9">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@BeforeMethod beforeMethod" time="-1.709064913635E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod beforeMethod -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@AfterMethod afterMethod" time="-1.70906491364E9">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@AfterClass afterClass" time="-1.709064913642E9">
    <skipped/>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@BeforeMethod beforeMethod" time="1.098">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.testcases.testNg.TestNgBase.beforeMethod(TestNgBase.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod beforeMethod -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@AfterMethod afterMethod" time="-1.709064914771E9">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@AfterClass afterClass" time="-1.709064914771E9">
    <skipped/>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@AfterTest afterTest" time="-1.709064914772E9">
    <skipped/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase classname="com.qa.testcases.testNg.TestNgClass" name="testa" time="0.0">
    <skipped/>
  </testcase> <!-- testa -->
  <testcase classname="com.qa.testcases.testNg.TestNgClass" name="testb" time="0.0">
    <skipped/>
  </testcase> <!-- testb -->
  <testcase classname="com.qa.testcases.testNg.TestNgClass" name="testc" time="0.0">
    <skipped/>
  </testcase> <!-- testc -->
  <testcase classname="com.qa.testcases.testNg.TestNgLogin" name="registration" time="0.002"/>
  <testcase classname="com.qa.testcases.testNg.TestNgLogin" name="login" time="0.001"/>
  <testcase classname="com.qa.testcases.testNg.TestNgLogin" name="search" time="0.001"/>
  <testcase classname="com.qa.testcases.testNg.TestNgLogin" name="search" time="0.0"/>
  <testcase classname="com.qa.testcases.testNg.TestNgLogin" name="search" time="0.0"/>
  <testcase classname="com.qa.testcases.testNg.TestNgLogin" name="exceptionHandling" time="0.001"/>
  <testcase classname="com.qa.testcases.testNg.TestNgLogin" name="logout" time="0.0"/>
  <testcase classname="com.qa.testcases.testNg.TestNgLogin" name="treminate" time="0.004">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.qa.testcases.testNg.TestNgLogin.treminate() didn&amp;apos;t finish within the time-out 1">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.qa.testcases.testNg.TestNgLogin.treminate() didn't finish within the time-out 1
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- treminate -->
  <testcase classname="com.qa.testcases.testNg.parameter.YahooLoginTest" name="loginToYahoo" time="0.0">
    <skipped/>
  </testcase> <!-- loginToYahoo -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@BeforeTest beforeTest" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest beforeTest -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@BeforeTest beforeTest" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest beforeTest -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@AfterTest afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@AfterTest afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@BeforeSuite beforeTestSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeTestSuite -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@BeforeSuite beforeTestSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeTestSuite -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@AfterSuite afterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
  <testcase classname="com.qa.testcases.testNg.TestNgBase" name="@AfterSuite afterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
</testsuite> <!-- Test -->
